backends/advanced/init.py

```
@@ -472,7 +457,7 @@ def show_summary(self):
         
         self.console.print(f"✅ Admin Account: {self.config.get('ADMIN_EMAIL', 'Not configured')}")
         self.console.print(f"✅ Transcription: {self.config.get('TRANSCRIPTION_PROVIDER', 'Not configured')}")
-        self.console.print(f"✅ LLM Provider: {self.config.get('LLM_PROVIDER', 'Not configured')}")
+        self.console.print(f"✅ LLM: Configured in config.yml (defaults.llm)")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)